debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/TablesWithoutPrimaryKeyIT.java

```diff
@@ -42,6 +42,28 @@ public void before() throws SQLException {
     public void shouldProcessFromSnapshot() throws Exception {
         TestHelper.execute(STATEMENTS);
 
+        start(PostgresConnector.class, TestHelper.defaultConfig()
+                .with(PostgresConnectorConfig.SNAPSHOT_MODE, SnapshotMode.INITIAL_ONLY)
+                .with(PostgresConnectorConfig.SCHEMA_INCLUDE_LIST, "nopk")
+                .build());
+        assertConnectorIsRunning();
+
+        final int expectedRecordsCount = 1 + 1 + 1;
```

Comment:
Do you see a big gain by making local vars final? I know @jpechane and I disagree on that, personally I think it's just noise and doesn't help really. Can you please come in my camp so we can outnumber @jpechane ;)

Reply:
immutability for the win! like this they can be for example directly been used inside lambda context.

Reply:
Which you don't do here, though :)